packages/evm/contracts/core/serialize/Integrity.sol

```diff
@@ -116,6 +116,19 @@ library Integrity {
             if (compValue.length != 0) {
                 revert IRolesError.UnsuitableCompValue(index);
             }
+        } else if (operator == Operator.Slice) {
+            // Slice can only be applied to a dynamic parameter
```

Comment:
it would make sense to also allow static (bytes32) params.

(we don't have to look farther than this very Roles contract for an example of something being tightly packed into bytes32 😉)

Reply:
Indeed